Azure/sonic-buildimage
# Added Nokia ixr7250E support

#### Why I did it
Support Nokia ixr7250E IMM and Supervisor cards
#### How I did it
Added modules x86_64-nokia_ixr7250e_sup-r0 and  x86_64-nokia_ixr7250e_36x400g-r0 ../device/nokia directory.
Modified the platform/broadcom/one-image.mk to include NOKIA_IXR7250_PLATFORM_MODULE 
Modified the platform/broadcom/rule.mk  to include the platform-module-nokia.mk 
#### How to verify it
Make sure the sonic-buildimage is successful
Install the image on device and verify the related components are installed and shown correctly. Such as the dpkg the correct package with the correct platform string 
#### Which release branch to backport (provide reason below if selected)
None



#### Description for the changelog
Added support for Nokia IXR7250E

#### A picture of a cute animal (not mandatory but encouraged)
